File: pset6/mario.py
import cs50
def main():
    while True:
        print("Height :",end="")
        h=cs50.get_int()
        if h<0 or h>23:
            break
        for i in range(h):
            for j in range(i+1,h):
                print(" ",end="")
            for k in range(2+i):
                print("#",end="")
            print("")
            
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include<cs50.h>
#include<stdio.h>
int main(void)
{
    int h,i,j,k;
    
    
    do
    {
        printf("Height: ");
        h=get_int();

            for(i=1;i<=h;i++)
            {
                for(j=i;j<h;j++)
                printf(" ");
                for(k=1;k<2+i;k++)
                printf("#");
                printf("\n");
            }
        
    }while(h<0||h>23);
}